src/encoded/batch_download.py

```diff
@@ -307,19 +307,21 @@ def batch_download(context, request):
     files = [metadata_link]
     if 'files.file_type' in param_list:
         for exp in results['@graph']:
-            for f in exp['files']:
-                if f['file_type'] in param_list['files.file_type']:
+            if 'files' in exp:
```

Comment:
I think for f in exp.get('files',[]) better than the if ... for.

Reply:
Sounds good. I made the changes.